src/code42cli/cmds/legal_hold.py

```diff
@@ -238,6 +263,50 @@ def _print_matter_members(username_list, member_type="active"):
         echo("No {} matter members.\n".format(member_type))
 
 
+def _merge_matter_members_with_devices(sdk, user_dataframe):
+    devices_generator = sdk.devices.get_all(active="true", include_backup_usage=True)
```

Comment:
py42 prefers a bool for the `active` param

Reply:
If for some reason the request fails in py42 when you do that, then this is fine, but I'd like to open a PR for py42 there if that is the case